src/equations/acoustic_perturbation_2d.jl

```diff
@@ -0,0 +1,102 @@
+@doc raw"""
+TODO
+"""
+struct AcousticPerturbationEquations2D{RealT<:Real} <: AbstractAcousticPerturbationEquations{2, 3}
+  v_avg::SVector{2, RealT}
+  rho_avg::RealT
+  c_sq_avg::RealT
+end
+
+function AcousticPerturbationEquations2D(v_avg::NTuple{2,<:Real}, rho_avg::Real, c_sq_avg::Real)
+  AcousticPerturbationEquations2D(SVector(v_avg), rho_avg, c_sq_avg)
+end
+
+
+get_name(::AcousticPerturbationEquations2D) = "AcousticPerturbationEquations2D"
+varnames(::typeof(cons2cons), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_prime")
+varnames(::typeof(cons2prim), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_prime")
+
+
+"""
+initial_condition_gauss(x, t, equation::AcousticPerturbationEquations2D)
+
+A Gaussian pulse.
+"""
+function initial_condition_gauss(x, t, equations::AcousticPerturbationEquations2D)
+  p_prime = exp(-4*(x[1]^2 + x[2]^2)) / 2
+  v1_prime = 1.0
+  v2_prime = 1.0
```

Comment:
I think these should be set to zero for this test
```suggestion
  v1_prime = 0.0
  v2_prime = 0.0
```